docs/pages/blog/mui-x-v6.md

```diff
@@ -0,0 +1,188 @@
+---
+title: MUI X v6 is ready for production!
+description: Introducing the new major version of the advanced components.
+date: 2023-03-06T00:00:00.000Z
+authors: ['josefreitas']
+card: true
+tags: ['MUI X', 'News']
+---
+
+<img src="/static/blog/mui-x-v6/card.png" alt="" style="width: 100%; margin-bottom: 16px;" />
+
+[MUI X v6.0.0](https://github.com/mui/mui-x/releases/tag/v6.0.0) is finally here with many improvements, new features, customization abilities, and a more robust foundation to accommodate the changes we want to deliver next.
+
+## ✨ Features highlights ✨
+
+### Data Grid
+
+#### [ApiRef now available for all users](/x/react-data-grid/api-object/)
+
+Manage pagination, scrolling, state, and other attributes through the Data Grid's API object—previously available only in commercial plans, now accessible to all.
+The ApiRef enables developers to implement a whole new range of customizations that rely on programmatic control of the Grid's features.
+
+```tsx
+function CustomDataGrid(props) {
+  const apiRef = useGridApiRef();
+
+  return (
+    <div>
+      <Button onClick={() => apiRef.current.setPage(0)}>
+        Go to page first page
+      </Button>
+      <DataGrid apiRef={apiRef} {...other} />
+    </div>
+  );
+}
+```
+
```

Comment:
Would it make sense to add a paragraph about why we are doing this move, making more open-source? We could also use the opportunity to link back to our stewardship https://mui-org.notion.site/Stewardship-542a2226043d4f4a96dfb429d16cf5bd.